styles/layouts/_main-navigation.scss

```diff
@@ -214,6 +219,10 @@
     font-weight: 500;
     cursor: pointer;
     border-radius: 0 !important;
+
+    &:hover {
+      color: $k-hovergrey !important;
+    }
```

Comment:
Could we remove these codes? Or it would make something different compared to line121~ line125?

Reply:
yeah that one seems to be a different class